<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timetable</class>
 <widget class="QWidget" name="timetable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>course mid:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="roomInput"/>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="ExitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Update Timetable</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="ConfirmButton">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>room information:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="timeInput"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>time information:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTextBrowser" name="textDisplay"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="midInput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
